### PR TITLE
21154 - Unit tests for all involuntary dissolution views/components

### PR DESCRIPTION
**Pull request by JazzarKarim**
*Issue #:*
https://github.com/bcgov/entity/issues/21154

*Description of changes:*
- Created/Updated the units tests for everything involuntary dissolution related

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Review by seeker25 — APPROVED** [review]

**Comment by seeker25** [comment]
image

File: auth-web/tests/unit/components/CardHeader.spec.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  beforeEach(() => {
+    const localVue = createLocalVue()
+
+    wrapper = mount(CardHeader, {
+      localVue,
+      vuetify: new Vuetify({}),
+      propsData: {
+        badgeText: 'Megatron',
+        icon: 'mdi-calendar-clock',
+        label: 'Optimus Prime',
```
Comment (JazzarKarim) [inline auth-web/tests/unit/components/CardHeader.spec.ts:17]: optimus-prime-1

😆

Comment (seeker25) [reply auth-web/tests/unit/components/CardHeader.spec.ts:17]: hahah

**Comment by JazzarKarim** [comment]
One sec, last commit didn't fix the SonarCloud issue. Trying something else now.

**Comment by JazzarKarim** [comment]
> image

Fixed! Thanks for spotting this Travis 💪

**Pull request merged**